src/components/navigation-bar/ui/auth/AuthBalance.tsx

```diff
@@ -0,0 +1,49 @@
+import { FC } from "react";
+import { useStyletron } from "baseui";
+import { PRIMITIVE_COLORS } from "../../../../shared";
+import { LabelSmall } from "baseui/typography";
+import { expandProperty } from "inline-style-expand-shorthand";
+
+export type AuthBalanceProps = {
+  amount: string;
+  coin?: string;
+  currency: string;
+};
+
+const containerStyles = {
+  ...expandProperty("padding", "16px"),
+  ...expandProperty("borderTop", `1px solid ${PRIMITIVE_COLORS.primary100}`),
+};
+
+const wrapperStyles = {
+  marginTop: "4px",
+};
+
+const labelStyles = {
+  marginRight: "4px",
+};
+
+const AuthBalance: FC<AuthBalanceProps> = ({ amount, coin, currency }) => {
+  const [css] = useStyletron();
+
+  return (
+    <div className={css(containerStyles)}>
```

Comment:
this container should be moved out from here to reuse it in another place will be possible. It should be really AuthItem. And current AuthItem should be AuthBlock in current scheme.